docs/customization/grid.rst

```diff
@@ -127,6 +127,67 @@ See an example of fields order modification on the `sylius_admin_product_review`
                     author:
                         position: 4
 
+Customize grids by events
+-------------------------
+
+There is also a second way to customize grids: **via events**.
+Every grid configuration dispatches an event when its definiton is converting.
+
+For example, **sylius_admin_product** grid dispatches an event:
+
+.. code-block:: php
+
+    sylius.grid.admin_product # For the grid of products in admin
+
+**1.** In order to remove fields from the product grid in **Sylius** you have to create a ``AppBundle\Grid\AdminProductsGridListener`` class.
```

Comment:
Before presenting the tutorial we should write what are we going to do:

`To show you and example of a grid customization using events, 
we will remove a field from a grid using that method. Here are the steps, that you need to take:`